siad/apipeer.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/NebulousLabs/Sia/network"
+)
+
+func (d *daemon) peerAddHandler(w http.ResponseWriter, req *http.Request) {
+	err := d.core.AddPeer(network.Address(req.FormValue("addr")))
```

Comment:
too much.

Daemon.Core.AddPeer(network.Address(req.FormValue("addr")))

That's two dereferences into a typecast into a function call.

```go
addr := network.Address(req.FormValue("addr"))
err := d.core.AddPeer(addr)
```